src/ship.cpp

```diff
@@ -29,16 +31,40 @@ bool Ship::collides(sf::Vector2f& pixelPos) const
     return this->radius < std::sqrt(dx * dx + dy * dy);
 }
 
-void Ship::trackMouse(float deg)
+void Ship::trackMouse(float rad)
```

Comment:
could be named better. trackMouse is what you will be using this function for but doesnt describe how it relates to the class itself